src/world/format/io/leveldb/LevelDB.php

```diff
@@ -199,6 +200,16 @@ protected function deserializeBlockPalette(BinaryStream $stream, \Logger $logger
 				$blockStateData = $this->blockDataUpgrader->upgradeBlockStateNbt($blockStateNbt);
 			}catch(BlockStateDeserializeException $e){
 				//while not ideal, this is not a fatal error
+				$id = $blockStateNbt->getString("name");
+				if ($id != null) {
+					$item = StringToItemParser::getInstance()->parse($id);
+					if ($item != null) {
+						//$palette[] = $this->blockStateDeserializer->deserialize(BlockStateData::current($id, []));
+						$palette[] = $item->getBlock()->getStateId();
+						$logger->error("Failed to upgrade blockstate: " . $e->getMessage() . " offset $i in palette, blockstate NBT: " . $blockStateNbt->toString() . ". Fixed!");
+						continue;
+					}
+				}
```

Comment:
Unrelated change - I'm not sure what exactly is going on here ...

Reply:
My mistake, removed by last commit